### PR TITLE
Move away from env files to a single env set at build time

### PR DESCRIPTION
**Pull request by depombo**
Also sets the sentry environment

### CODE REVIEW

File: src/config/production.ts [hunk 1]
```diff
@@ -2,16 +2,18 @@ import { ConfigInterface, throwError, } from './config';
 
 const config: ConfigInterface = {
   port: 8088,
-  dbHost: process.env.DB_HOST ?? throwError('No DB Host defined'), // TODO: Move away from env var
+  dbHost: 'https://db.iasql.com',
```
Comment (dfellis) [inline src/config/production.ts:5]: Drop the http part

Comment (dfellis) [reply src/config/production.ts:5]:
```suggestion
  dbHost: 'db.iasql.com',
```

File: src/config/staging.ts [hunk 2]
```diff
@@ -2,16 +2,18 @@ import { ConfigInterface, throwError, } from './config';
 
 const config: ConfigInterface = {
   port: 8088,
-  dbHost: process.env.DB_HOST ?? throwError('No DB Host defined'), // TODO: Move away from env var
+  dbHost: 'https://db-staging.iasql.com',
```
Comment (dfellis) [inline src/config/staging.ts:5]:
```suggestion
  dbHost: 'db-staging.iasql.com',
```

**Comment by dfellis** [comment]
To make sure that we don't forget. I think the `IASQL_ENV` var needs to be declared in the Dockerfile, too.

**Review by dfellis — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -10,8 +10,8 @@
     "test": "IASQL_ENV=test jest -i",
     "coverage": "IASQL_ENV=test jest -i --coverage",
     "coverage:local": "IASQL_ENV=test export $(cat .env | xargs) && yarn coverage",
-    "start": "if [ \"$IASQL_ENV\" = \"\" ]; then IASQL_ENV=production yarn serve; elif [ \"$IASQL_ENV\" = \"local\" ]; then yarn start:dev; else yarn serve; fi",
```
Comment (aguillenv) [inline package.json:13]: removing this breaks the `docker-compose:dev` script to start locally with hot reload. I'll see how can we could maintain it

**Pull request merged**